importScripts/refineryParameters/Conservation.ca_occurrences.conservation.occurrenceSplitter.json

```diff
@@ -1,13 +1,15 @@
 {
     "occurrenceType": "Conservation",
     "relationshipType": "conservation",
-    "name": "^Conservation",
     "matchOn": [
         "idno"
     ],
-    "attributes":{
+    "attributes": {
         "ConservationDetails": "^Conservation",
         "ConservationDate": "^ConservationDate",
-        "Conservator": "^ConservationBy"
+        "Conservator": "^ConservationBy",
+        "preferred_labels":{
+            "name": "^Accession_Full_ID conserved on ^ConservationDate"
```

Comment:
We don't need the conditional template here, as all source values have an object associated with them.